packages/dx-react-demos/src/demo-viewer/demo-renderer.jsx

```diff
@@ -15,11 +17,19 @@ export const DemoRenderer = ({
   const { DemoContainer } = themes
     .find(({ name: theme }) => theme === currentTheme);
 
-  return (
+  const demo = (
     <DemoContainer>
       <Component />
     </DemoContainer>
   );
+  if (currentTheme === 'material-ui') {
```

Comment:
Why do you write it here? There is DemoContainer for that purposes.

Reply:
In this case, I wrap all elements in a demo with Paper. It makes sense for the Theming demo.